src/parser/jobs/dnc/changelog.tsx

```diff
@@ -1,105 +1,10 @@
-import CONTRIBUTORS from 'data/CONTRIBUTORS'
-import React from 'react'
+// import CONTRIBUTORS from 'data/CONTRIBUTORS'
+// import React from 'react'
 
 export const changelog = [
 	// {
 	// 	date: new Date('2021-11-19'),
 	// 	Changes: () => <>The changes you made</>,
-	// 	contrubutors: [CONTRIBUTORS.YOU],
+	// 	contributors: [CONTRIBUTORS.YOU],
```

Comment:
lel

Reply:
It bothered me since last expac and just didn't do anything about it then 🙃